### PR TITLE
Add diffoscope

### PR DESCRIPTION
**Pull request by bollwyvl**
Adds https://diffoscope.org/

Dependent on https://github.com/conda-forge/staged-recipes/pull/3281

### CODE REVIEW

**Comment by bollwyvl** [comment]
Huzzah, looking good.

File: recipes/diffoscope/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (dougalsutherland) [inline recipes/diffoscope/meta.yaml]: Why do you have both a `script` entry and a `build.sh`? You could delete `build.sh` and do

```yaml
  script: PYTHONHASHSEED=0 python setup.py install --single-version-externally-managed --record record.txt
```

Comment (bollwyvl) [inline recipes/diffoscope/meta.yaml]: Thanks, good catch!

**Comment by bollwyvl** [comment]
Anything else needed here?

**Comment by msarahan** [comment]
Thanks @bollwyvl and @dougalsutherland

**Pull request merged**